Packs/TAXIIServer/Integrations/TAXII2Server/README.md

```diff
@@ -23,37 +23,48 @@ You can add a collection description as is done in *collection1_name*, or enter
+  For example, `ext-xsoar.mycompany.com`.  
+- The instance name cannot be changed after saving the integration configuration.
 
 
 ## Access the TAXII Service by Instance Name
 
-To access the TAXII service by instance name, make sure *Instance execute external* is enabled. 
+To access the TAXII service by instance name, make sure ***Instance execute external*** is enabled. 
 
 In Cortex XSOAR 6.x:
 1. Navigate to **Settings > About > Troubleshooting**.
-2. In the **Server Configuration** section, verify that the *instance.execute.external* key is set to *true*. If this key does not exist, click **+ Add Server Configuration**, add the *instance.execute.external* and set the value to *true*.
+2. In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to **true**. If this key does not exist, click **+ Add Server Configuration**, add the ***instance.execute.external*** and set the value to **true**.
 
-### How to use HTTPS
+## How to Use HTTPS
 
 To use HTTPS, a certificate and private key have to be supplied in the integration configuration. 
 
-### How to use authentication
+## Set up Authentication
+### For Cortex XSOAR 8 Cloud Tenant or Cortex XSIAM Tenant
+The TAXII2 Server integration running on a Cortex XSOAR 8 Cloud tenant or Cortex XSIAM tenant enables using basic authentication in the requests.  
+To enable basic authentication, a user and password must be supplied in the **Credentials** parameters in the integration configuration.  
+The server will then authenticate the requests by the `Authorization` header, expecting basic authentication encrypted in base64 to match the given credentials.  
+### For Cortex XSOAR On-prem (6.x or 8) or When Using Engines
+For Cortex XSOAR On-prem (6.x or 8) or when using engines, you can set up authentication using custom certificates. For more information on setting up a custom certificate for Cortex XSOAR 8 On-prem, see [HTTPS with a signed certificate](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/HTTPS-with-a-signed-certificate). For more information on setting up a custom certificate for Cortex XSOAR 6.x, see [HTTPS with a Signed Certificate](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.13/Cortex-XSOAR-Administrator-Guide/HTTPS-with-a-Signed-Certificate).
 
-The integration allows the use of basic authentication in the requests.
-To enable basic authentication, a user and password must be supplied in the *Credentials* parameters in the integration configuration.
+## TAXII v2.0 API Enpoints
 
-The server will then authenticate the requests by the *Authorization* header, expecting basic authentication encrypted in base64 to match the given credentials.
+**Note**:  
+For Cortex XSOAR 8 On-prem, you need to add the `api-` FQDN DNS record that maps the Cortex XSOAR DNS name to the API IP address.  
+For example, `api-xoar.mycompany.com`.
```

Comment:
What is this?

Reply:
I added it based on input from @Bharathwaj-Murali _For the integrations, we need to add this note to Core REST API as well since that uses api-FQDN._
Is it not relevant here?

Reply:
@julieschwartz18 don't understand why this is relevant here, we are talking on a TAXIIServer integration not something that is using the public API

Reply:
@ShahafBenYakir - ok, got it. I'm opening a new PR to remove it